src/sentry/static/sentry/app/data/platformCategories.tsx

```diff
@@ -41,6 +45,8 @@ const mobile = [
 
 const backend = [
   'csharp',
+  'dotnet',
+  'dotnet.aspnetcore',
```

Comment:
what's this dot thing

Reply:
oops changing - but thats the format we use in docs itself